### PR TITLE
Updates related to new pricing

### PR DESCRIPTION
**Pull request by chexton**

### CODE REVIEW

File: _articles/what-happens-if-you-exceed-your-subscription-limits.md [hunk 1]
```diff
@@ -6,10 +6,42 @@ categories:
...
+
+Once you lower your message volume or delete customer records you can downgrade your plan using our UI.
+
+You can view your usage and manage your plan under [Account > Billing](https://app.getvero.com/account/billing).
+
+## For customers who signed up after June 2020
+
+As well as specific features, each plan includes a total number of user profiles, a monthly message limit and a monthly event tracking limit. 
+
+You can manually choose the plan that best suits the features and team access you need. We will not automatically move you between our four pricing plans.
```
Comment (probablyrory) [inline _articles/what-happens-if-you-exceed-your-subscription-limits.md:25]: You've said "team access" here, but we don't do that yet, I'd suggest changing to say "choose the plan that best suits the features you need, and the number of team members that require access."

File: _articles/what-happens-if-you-exceed-your-subscription-limits.md [hunk 2]
```diff
@@ -6,10 +6,42 @@ categories:
...
+
+If you exceed the included usage limits within a tier we will charge you overage for your extra usage, rounded to the nearest thousand users. To keep things simple, we charge overage based on the one, "customers", metric. Each of Vero's plans defines an "overage rate" per thousand customer records. Every thousand customer records also includes an extra quota of messages and events tracked. 
+
```
Comment (probablyrory) [inline _articles/what-happens-if-you-exceed-your-subscription-limits.md:28]: I'd include the number of emails and events tracked per customer here before you dive into the example.

File: _articles/what-happens-if-you-exceed-your-subscription-limits.md [hunk 3]
```diff
@@ -6,10 +6,42 @@ categories:
...
+## For customers who signed up prior to June 2020
 
-You can downgrade at any time and, if your usage fluctuates, can minimize your spend each and every month.
+Customers who signed up prior to June 2020 are grandfathered on the plans they chose when they first paid.
 
-You can manage your subscriptions in the Vero under **Account > Billing**.
\ No newline at end of file
+Vero will send you an email when you cross the customer or message thresholds over a rolling 30 day window. You will automatically be upgraded to the next tier, based on your increased usage. As a result, invoices are charged in arrears. You are charged at the "high watermark" for a given month, as upgrades reflect usage that has already occurred.
+
+Once you lower your message volume or delete customer records you can downgrade your plan using our UI.
```
Comment (probablyrory) [inline _articles/what-happens-if-you-exceed-your-subscription-limits.md:17]: AFAIK you can't downgrade in the UI.

**Comment by probablyrory** [comment]
@chexton I think I've finished it. Might be worth a quick review though.

File: _articles/what-happens-if-you-exceed-your-subscription-limits.md (hunk 1 above)
Comment (chexton) [reply _articles/what-happens-if-you-exceed-your-subscription-limits.md]: @probablyrory I think we do support "team access" don't we? I.e. 1, 5, 20 user seats?

File: _articles/what-happens-if-you-exceed-your-subscription-limits.md (hunk 3 above)
Comment (chexton) [reply _articles/what-happens-if-you-exceed-your-subscription-limits.md:17]: @probablyrory currently you can't...interesting. I thought you could. Could you change this to "by getting in touch via support@getvero.com" please.

File: _articles/what-happens-if-you-exceed-your-subscription-limits.md (hunk 2 above)
Comment (chexton) [reply _articles/what-happens-if-you-exceed-your-subscription-limits.md:34]: Good idea, could you add in :)?

File: _articles/what-happens-if-you-exceed-your-subscription-limits.md (hunk 1 above)
Comment (probablyrory) [reply _articles/what-happens-if-you-exceed-your-subscription-limits.md]: I guess you're right, I was thinking it meant user permissions.

File: _articles/what-happens-if-you-exceed-your-subscription-limits.md (hunk 3 above)
Comment (probablyrory) [reply _articles/what-happens-if-you-exceed-your-subscription-limits.md:17]: Already done 👍

**Pull request merged**